salt/_modules/containerd.py

```diff
@@ -0,0 +1,60 @@
+'''
+Various functions to interact with :program:`containerd` using :command:`ctr`.
+'''
+
+import re
+import logging
+
+import salt.utils.path
+
+log = logging.getLogger(__name__)
+
+
+__virtualname__ = 'containerd'
+
+
+def __virtual__():
+    if not salt.utils.path.which('ctr'):
+        return (False, '\'ctr\' command not found')
```

Comment:
Use double quote when the string contains single quote: it's more legible than escaping (or could also use backtick instead of single quote for that purpose).

Also parenthesis are usually omitted here: `return False, "'ctr' command not found"`

Reply:
Same as the quotes remark, it's how it's done in other Salt modules.

Reply:
Oh, forgot: single-quotes, will do.